### DISCUSSION TITLE
protocol.h Issue

### DISCUSSION THREAD

**Discussion by ApacheBlack** [jnsw/bt2bp#5] [General]
Hello! 
I am having problem compiling with VS.  i am having this all the time. what could be the solution?
thanks. 
 #include "protocol.h"
          ^~~~~~~~~~~~
compilation terminated.
Compiling .pio\build\control_PPM\src\control.o
In file included from src\bldc.h:3:0,
                 from src\bldc.c:6:
src\control_structures.h:24:10: fatal error: protocol.h: No such file or directory

**Comment by jnsw** [comment]
I am sorry, this project is kinda outdated, because the ArduinoController app from PlayStore is offline. But if you want to try nonetheless, then it seems the file is missing, maybe there is something broken on download or the libraries are outdated also.

**Reply by ApacheBlack** [reply]
ok thanks! any advice or good info you could share with me to help me to control the board with bluetooth,? the i can make it wit =h MIT APP inventor. thanks

**Reply by jnsw** [reply]
no i dont know, you can walk through my code and you will see the controller will just send bluetooth serial values for each axis and you have to interpret them to correctly to drive the motor, i am just using two different libraries, one from the ardu-control and the other from bipropellant -> but you can use the apk-pure download of the apk if you want to use this project
